src/traits.md

```diff
@@ -404,14 +384,13 @@ fn inverse<T>() -> T
 }
 ```
 
-This shows off the additional feature of `where` clauses: they allow bounds
-on the left-hand side not only of type parameters `T`, but also of types (`i32` in this case). In this example, `i32` must implement
-`ConvertTo<T>`. Rather than defining what `i32` is (since that's obvious), the
-`where` clause here constrains `T`.
+這展示 `where` 額外的特性，它們除了型別參數 `T`，亦可限制型別（範例中的 `i32`)
+在範例中，`i32` 必須實作 `ConvertTo<T>`，這裡的 `where` 限制 `T`而不是定義 `i32`是什麼
```

Comment:
I am also confused at `Rather than defining what i32 is (since that's obvious), the where clause here constrains T.`